src/sdk/http/http-client.ts

```diff
@@ -99,12 +99,22 @@ export abstract class HttpClient<TRequest, TResponse> implements IHttpClient {
 		}
 	}
 
+	static getFilteredParameters(params?: any): any {
```

Comment:
I don't think this needs to be a `static` method attached to `HttpClient`. I would move it up so it isn't a part of the public api.

Reply:
will do